client/components/Message/Attachments/components/Image.tsx

```diff
@@ -32,7 +32,7 @@ export const Retry: FC<BoxProps & { retry: () => void }> = ({ retry, ...props })
             background: var(--rxc-color-neutral-300, ${ colors.n300 }) !important;
         }
     `;
-	return <ImageBox className={clickable} {...props} onClick={retry}>
+	return <ImageBox className={clickable} {...props} onClick={retry} height='200px'>
```

Comment:
The design team of rocket.chat decided to use fixed sizes for Retry component, so I suggest to use the `size={160}` property in this component insted of `height='...'`.

Reply:
Done :)